src/java.base/share/classes/sun/security/provider/certpath/AdjacencyList.java

```diff
@@ -125,7 +125,7 @@ private boolean buildList(List<List<Vertex>> theList, int index,
 
         // Each time this method is called, we're examining a new list
         // from the global list. So, we have to start by getting the list
-        // that contains the set of Vertexes we're considering.
+        // that contains the set of Vertices we're considering.
```

Comment:
The class being affected is a Vertex, so either change to vertices, or leave as is...

Reply:
Agree, suggest changing it to "vertices".